src/modules/stackables/components/NotificationItem.js

```diff
@@ -18,8 +27,7 @@ const Update_Is_Fixed = 'update_is_fixed';
 export class NotificationItem extends MvuElement {
```

Comment:
No test for the default onClose function
See:
https://github.com/ldbv-by/bav4-nomigration/blob/39ce8e6f8c5141fe3c1be922c29326db4dacb971/test/modules/auth/components/PasswordCredentialPanel.test.js#L46